client/js/components/procedure/admin/InstitutionTagManagement/InstitutionList.vue

```diff
@@ -138,20 +162,36 @@ export default {
       institutionTagList: 'items'
     }),
 
+    ...mapState('InstitutionTagCategory', {
+      institutionTagCategories: 'items'
+    }),
+
     ...mapState('InvitableInstitution', {
       invitableInstitutionList: 'items',
       currentPage: 'currentPage',
       totalPages: 'totalPages'
     }),
 
-    tagList () {
-      return Object.values(this.institutionTagList).map(tag => {
-        const { id, attributes } = tag
-        return {
-          id,
-          name: attributes.name
-        }
-      })
+    categoryFieldsAvailable() {
+      return Object.values(this.institutionTagCategories).map(category => ({
```

Comment:
I found 4 occurrences of `Object.values(this.institutionTagCategories)` maybe we can move it to an computed?

Reply:
Good idea, done with [33bf4dc](https://github.com/demos-europe/demosplan-core/pull/4115/commits/33bf4dc1f15d5db21a87c5803189cc64bc909f18)